ai21_tokenizer/jurassic_tokenizer.py

```diff
@@ -52,6 +57,20 @@ def __init__(
         self._space_mode = config.get("space_mode")
         self._space_tokens = self._map_space_tokens()
 
+    @classmethod
+    def from_file_handle(
+        cls, model_file_handle: BinaryIO, config: Optional[Dict[str, Any]] = None
+    ) -> JurassicTokenizer:
+        return cls(model_path=None, model_file_handle=model_file_handle, config=config)
+
+    @staticmethod
+    def _assert_exactly_one(model_path: Optional[PathLike], model_file_handle: Optional[BinaryIO]) -> None:
```

Comment:
- We don't really use `staticmethod` I prefer this would be a regular protected method of the class
- rename to `_validate_init`

Reply:
Interesting. Why not use staticmethod if the method doesn't need `self`?